package.json

```diff
@@ -14,7 +14,6 @@
   },
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230719670-1d582df0-b008-4ce4-a545-4c169f481394.png)

- [ ] Great work on implementing the tests so far :+1: but i guess you forgot about this requirement in your project that you will need to stimulate the user interaction such as clicking a button and see the output by clicking that button :+1: Kindly add fireEvents in your tests as well to follow the requirements :+1: